### DISCUSSION TITLE
AC or DC

### DISCUSSION THREAD

**Discussion by nliaudat** [nliaudat/esp32_8ch_motor_shield#16] [Polls]
Hi, 
I work on migrating the AC power supply to DC input only

![Capture](https://user-images.githubusercontent.com/6782613/189535494-2030a5c6-b53b-4315-a2cb-c4a32a0c1c82.PNG)

The DC input will be from DC 4.9V to 18V with a  DC Power Receptacle diam 2.5/6.3mm or an USB-C input

The why is that actual AC power would need some more component (fuse, big resistance and capacitor) to be safe.

![Capture](https://user-images.githubusercontent.com/6782613/189535732-68bb5196-0654-420b-a57c-1ce7a9c3f109.PNG)

The actual AC converter costs : 

HKL-5M03 : 2.3$
HKL-PM03 : 1.9 $

The new DC input power costs approx 1.3$ with converter, usb-c and DC plugs and capacitors

![Capture](https://user-images.githubusercontent.com/6782613/189536557-082be6a7-045b-4e5f-b878-b08ebfe7910c.PNG)

**Comment by Lonestar78** [comment]
Certainly safer to work with DC only on the board. Then use a good external powersupply. should then also be sufficient easily to run at least 4 actuators in parallel

**Comment by nliaudat** [comment]
new version for DC available at https://github.com/nliaudat/esp32_8ch_motor_shield/tree/main/pcb/DC
